en/0x11-V2-Software_Bill_of_Materials.md

```diff
@@ -3,7 +3,7 @@
 ## Control Objective
 
 Automatically creating accurate Software Bill-of-Materials (SBOM) in the build pipeline is one indicator of mature
-development processes. Generated SBOMs should ideally use a machine readable format such as CycloneDX, SPDX, or SWID. 
+development processes. SBOMs should be a machine readable format such as CycloneDX, SPDX, or SWID. 
```

Comment:
Change to:
"SBOMs should be in a machine readable format."  Drop the formats from the sentence.

Reply:
Apparently I combined two branches. Closing this one.